.travis.yml

```diff
@@ -12,33 +12,32 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
```

Comment:
only files is usefull to cache, the other directories contain files that frequently (every hour) change. Caching them will only reduce test speed.